common/v2/features/SendAssets/components/SendAssetsForm.tsx

```diff
@@ -244,10 +244,12 @@ export default function SendAssetsForm({ txConfig, onComplete }: IStepComponentP
   });
 
   const validAccounts = accounts.filter(account => account.wallet !== WalletId.VIEW_ONLY);
+  const userAccountEthAsset = userAssets.find((a: Asset) => a.ticker === 'ETH');
```

Comment:
You can replace this with 
`const userAccountEthAsset = userAssets.find((a: Asset) => a.uuid === EtherUUID);`